typescript-rest/src/options.ts

```diff
@@ -1,5 +1,5 @@
 import { Agent } from "http";
-import { Headers } from "node-fetch";
+// import { Headers } from "node-fetch";
```

Comment:
should remove unused code instead of comment off

Reply:
removed